File: Python/primeno.py
# num=int(input("enter a num :"))

# if(num<=1):
#      print("not prime number")
# else:
#     for i in range(2,int(num**0.5)+1):
#       if(num % i == 0):
#          print("not prime")
#          break
#     else:
#         print("Prime num")
    
num=int(input("enter your num :"))
flag=0
for i in range(2,num//2 + 1):
    if(num % i==0):
       flag=1
       break

if(flag==0 and num>1):
      print("it is prime")
else:
      print("not prinme")

File: Java/primeno.java

import java.util.Scanner;

class primeno{
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the num :");
        int num=sc.nextInt();
        int flag=0;
        int i;

       if(num<=1){
        flag=1;
       } else{
                 for(i =2;i<=num/2;i++){
            if(num % i==0){
                flag=1;
                break;
            }
        }  
    }  
    if(flag ==0){
                System.out.println("It is prime");
    }else{
        System.out.println("Not prime");
    }   
    }
}  
    
